src/libraries/System.Private.CoreLib/src/System/Activator.RuntimeType.cs

```diff
@@ -136,10 +136,12 @@ public static partial class Activator
 
         [Intrinsic]
         public static T CreateInstance<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] T>()
+            // where T : allows ref struct // TODO: https://github.com/dotnet/runtime/pull/102636
```

Comment:
@AaronRobinsonMSFT, if your PR merges first, I'll fix this up, otherwise you can fix it up in yours.